JS/Lesson34/gulcin/recipe-app/src/App.js

```diff
@@ -1,23 +1,35 @@
 import { useState, useEffect } from 'react';
 import { Search } from './modules/recipes/SearchBar';
+import { RecipeDetails } from './modules/recipes/RecipeDetails';
 import './App.css';
 
 export const App = () => {
 	const [recipes, setRecipes] = useState([]);
+	const [id, setId] = useState('');
 
-	useEffect(() => {
+	function handleClick(e) {
+		const id= e.target.getAttribute('id');
+		setId(id);
+		
+	}
+	
+
+	/* useEffect(() => {
```

Comment:
Remove commented out code, it's not used.

Reply:
Done!